external/leveldb/LevelDbTuning.ipynb

```diff
@@ -714,20 +279,17 @@
     "    latency, throughput = float(stats[0].strip().split(\" \")[0]), float(stats[1].strip().split(\" \")[0])\n",
     "    return latency, throughput\n",
     "\n",
-    "#optimizer = bayesian_optimizer_factory.create_remote_optimizer(optimization_problem=optimization_problem)\n",
```

Comment:
There's another line wrap issue in `result = ` line of the the `run_workload` def above, but it's outside of this PR's diff so I can't suggest a change directly here.

Reply:
Fixed this one.